.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -1,8 +1,8 @@
 ---
 name: "\U0001F41B Bug report"
-about: To help the developer act on the issues, please include a description of your environment, preferably a minimum script to reproduce the problem.
+about: To help the developer act on the issues, please include a description of your environment, preferably a minimum script to reproduce the problem. Find the list of label tags at TBD.
 title: "[Bug] "
-labels: "type: bug"
+labels: "needs-triage"
```

Comment:
we can have multiple labels by default, should we keep the category (here and elsewhere)?
```suggestion
labels:
 - "needs-triage"
 - "type: bug"
```

Reply:
made changes to add back the default labels for the templates